app/main.py

```diff
@@ -1,3 +1,51 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open(
+        os.path.join(os.path.dirname(__file__), "config.json"), "r"
+    ) as config_file:
+        config = json.load(config_file)
```

Comment:
close stream after this part
just move code below 1 level down

Reply:
cant get it, may i ask u to explain?